content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -1,47 +1,194 @@
+
+- :guilabel:`Helpdesk Team`: a policy can only be applied to one team. *This field is required.*
+- :guilabel:`Minimum Priority`: the priority level for a ticket is identified by selecting one, two,
+  or three of the :guilabel:`⭐ (star)` icons, representing the priority level on the Kanban card or
+  on the ticket itself. The :abbr:`SLA (Service Level Agreement)` is **only** applied after the
+  priority level has been updated on the ticket to match the :abbr:`SLA (Service Level Agreement)`
+  criteria. If no selection is made in this field, this policy applies to all priority levels.
+- :guilabel:`Type`: ticket types can be helpful when indicating when a ticket is a customer
+  question, that can be solved with a quick response, or an issue that may require additional
+  investigation. Multiple ticket types can be selected for this field. If no selection is made, this
+  policy applies too all ticket types.
+- :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied
+  to a single ticket.
+- :guilabel:`Customers`: individual contacts or companies may be selected in this field.
+- :guilabel:`Sales Order Items`: this field is available only if a team has the *Timesheets* feature
+  enabled. This allows the ticket to link directly to a specific line on a sales order, which must
+  be indicated on the ticket in the :guilabel:`Sales Order Items` field.
+
+Establish an SLA policy target
+------------------------------
+
+The *target* section identifies the stage a ticket needs to reach, and the time allotted to reach
+that stage, to satisfy the :abbr:`SLA (Service Level Agreement)` policy. Any stage assigned to a
+team may be selected for the :guilabel:`Reach Stage` field.
+
+.. example::
+   An :abbr:`SLA (Service Level Agreement)` titled `8 Hours to Close` tracks the working time before
+   a ticket is completed, and would have `Solved` as the :guilabel:`Reach Stage`. Simultaneously, an
+   :abbr:`SLA (Service Level Agreement)` titled `2 Days to Start` tracks the working time before
+   work on a ticket has begun, and would have `In Progress` as the :guilabel:`Reach Stage`.
+
+Time spent in stages selected in the :guilabel:`Excluding Stages` field are **not** included in the
+calculation of the :abbr:`SLA (Service Level Agreement)` deadline.
+
+Meet SLA deadlines
+==================
+
+As soon as it is determined that a ticket fits the criteria of an :abbr:`SLA (Service Level
+Agreement)` policy, a deadline is calculated. The deadline is based on the creation date of the
+ticket, and the targeted working hours.
+
+.. note::
+   The value indicated next to the :guilabel:`Working Hours` field of an :abbr:`SLA (Service Level
+   Agreement)` policy is used to determine the deadline. By default, this is determined by the value
+   set in the :guilabel:`Company Working Hours` field under :menuselection:`Settings app -->
+   Technical --> Resources --> Working Times`.
```

Comment:
correct me if I'm wrong, but you have to be in Developer Mode to see the "Technical" menu, right?
```suggestion
   Technical --> Resource section --> Working Times`.
```